testing/scenario_app/lib/src/platform_view.dart

```diff
@@ -712,10 +712,13 @@ mixin _BasePlatformViewScenarioMixin on Scenario {
     if (Platform.isIOS) {
       sceneBuilder.addPlatformView(viewId, width: width, height: height);
     } else if (Platform.isAndroid) {
+      const Offset offset = Offset(50, 50);
       if (usesAndroidHybridComposition) {
+        sceneBuilder.pushOffset(offset.dx, offset.dy);
+        sceneBuilder.addPlatformView(viewId, width: width, height: height);
```

Comment:
could we push a `clipRRect` here?

Reply:
why did we need to push clip rrect here? This is a general code for all scenarios.